react/ui-components/src/molecules/stories/SuccessPanelCard.stories.js

```
@@ -0,0 +1,252 @@
+  },
+  footerStyles:{}
+};
+
+const footerChildrenWithTwoButtons = [
+  <Button
+    type={"button"}
+    size={"large"}
+    variation={"secondary"}
+    label="Button"
+    onClick={() => console.log("Clicked Button 1")}
+  />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)